### PR TITLE
Update OPT conversion script to work for OPT-IML

### PR DESCRIPTION
**Pull request by thomasw21**

### CODE REVIEW

File: src/transformers/models/opt/convert_opt_original_pytorch_checkpoint_to_pytorch.py [hunk 1]
```diff
@@ -52,6 +52,27 @@ def load_checkpoint(checkpoint_path):
...
+            value = sd[key]
+            # We split QKV in seperate Q,K,V
+
+            q_name = key.replace(".qkv_proj.",".q_proj.")
+            k_name = key.replace(".qkv_proj.", ".k_proj.")
+            v_name = key.replace(".qkv_proj.", ".v_proj.")
+
+            depth = value.shape[0]
+            assert depth % 3 == 0
+            # `SequeuceParallelTransformerBlock` has QKV weight is seperated in K,V,Q despite the naming:
```
Comment (sgugger) [inline src/transformers/models/opt/convert_opt_original_pytorch_checkpoint_to_pytorch.py:68]:
```suggestion
            # `SequenceParallelTransformerBlock` has QKV weight is separated in K,V,Q despite the naming:
```

**Review by sgugger — APPROVED** [review]
Works for me once `make style` is applied.

File: src/transformers/models/opt/convert_opt_original_pytorch_checkpoint_to_pytorch.py (hunk 1 above)
Comment (thomasw21) [reply src/transformers/models/opt/convert_opt_original_pytorch_checkpoint_to_pytorch.py:68]: Actually there's a typo in `metaseq` so it really is `SequeuceParallelTransformerBlock`

Comment (thomasw21) [reply src/transformers/models/opt/convert_opt_original_pytorch_checkpoint_to_pytorch.py:68]:
```suggestion
            # `SequeuceParallelTransformerBlock` has QKV weight is separated in K,V,Q despite the naming:
```

**Pull request merged**